File: TimeToCode/app/src/main/java/com/kmu/timetocode/Support.java
package com.kmu.timetocode;

import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.Button;
import android.widget.ImageButton;

import androidx.annotation.Nullable;
import androidx.fragment.app.Fragment;

public class Support extends Fragment{

    public static Support newInstance(){
        return new Support();
    }
    @Override
    public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container,
                             @Nullable Bundle savedInstanceState) {
        ViewGroup support = (ViewGroup) inflater.inflate(R.layout.fragment_support, container, false);
        ImageButton backButton = support.findViewById(R.id.backCertification);
        ImageButton questionButton = support.findViewById(R.id.questionButton);
        ImageButton inquiryButton = support.findViewById(R.id.inquiryButton);
        backButton.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                ((NavActivity)getActivity()).replaceFragment(MainFragment.newInstance());
            }
        });
        questionButton.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                ((NavActivity)getActivity()).replaceFragment(Question.newInstance());
            }
        });
        inquiryButton.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                ((NavActivity)getActivity()).replaceFragment(Inquiry.newInstance());
            }
        });
        return support;
    }
}

File: TimeToCode/app/src/main/java/com/kmu/timetocode/Support.kt
package com.kmu.timetocode

import android.view.LayoutInflater
import android.view.ViewGroup
import android.os.Bundle
import android.view.View
import android.widget.ImageButton
import androidx.fragment.app.Fragment

class Support : Fragment() {
    override fun onCreateView(
        inflater: LayoutInflater, container: ViewGroup?,
        savedInstanceState: Bundle?
    ): View? {
        val support = inflater.inflate(R.layout.fragment_support, container, false) as ViewGroup
        val backButton = support.findViewById<ImageButton>(R.id.backCertification)
        val questionButton = support.findViewById<ImageButton>(R.id.questionButton)
        val inquiryButton = support.findViewById<ImageButton>(R.id.inquiryButton)
        backButton.setOnClickListener { (activity as NavActivity?)!!.replaceFragment(MainFragment()) }
        questionButton.setOnClickListener { (activity as NavActivity?)!!.replaceFragment(Question.newInstance()) }
        inquiryButton.setOnClickListener { (activity as NavActivity?)!!.replaceFragment(Inquiry.newInstance()) }
        return support
    }

    companion object {
        fun newInstance(): Support {
            return Support()
        }
    }
}